src/pages/collection/[address].tsx

```diff
@@ -145,17 +190,128 @@ export default function Collection() {
     bg-white border border-gray-200 rounded-lg shadow-md 
     dark:bg-gray-800 dark:border-gray-700 max-w-full max-h-full
     `}>
-      <Line data={data} width={400} height={300} options={options}/>
+
+      <div className="float-right p-2 space-x-2">
+        <select 
+          className="rounded-md p-2 bg-slate-100 hover:bg-slate-200 dark:bg-slate-700 dark:text-slate-300 dark:hover:bg-slate-600 shadow-md"
+          value={selector} 
+          onChange={patchedHandleSelect}
+          >
+          {dataOptions.map((o) => {
+            return <option key={o} value={o}>{o}</option>
+          })}
+        </select>
+        <select 
+          className="rounded-md p-2 bg-slate-100 hover:bg-slate-200 dark:bg-slate-700 dark:text-slate-300 dark:hover:bg-slate-600 shadow-md"
+          value={records} 
+          onChange={(e: any & {target: {value: number}}) => setRecords(e.target.value)}
+          >
+          <option key={7} value={7}>7d</option>
+          <option key={30} value={30}>30d</option>
+          <option key={365} value={365}>1y</option>
+        </select>
+      </div>
+
+      <div className="px-4">
+        <LineGraph width={800} height={400} yCallback={yAxisCallback} graphLabel={selector} labels={graphLabels} dataPts={dataPts}/> 
+      </div>
     </div>
 
 
   return (
-    <div className="py-4 grid grid-cols-1 lg:grid-cols-2 gap-x-4 gap-y-4 grid-rows-2 lg:grid-rows-1 h-full">
+    <div className="grid grid-cols-1 lg:grid-cols-2 gap-x-4 gap-y-4 grid-rows-2 lg:grid-rows-1">
```

Comment:
We could center the graph further down so it won't look so empty on the bottom margin.

Reply:
fixed